app/org/sagebionetworks/bridge/models/ReportTypeResourceList.java

```diff
@@ -7,23 +7,19 @@
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
-public class ReportTypeResourceList<T> {
-    private final List<T> items;
-    private final ReportType reportType;
-
+public class ReportTypeResourceList<T> extends ResourceList<T> {
+    
     @JsonCreator
-    public ReportTypeResourceList(@JsonProperty("items") List<T> items,
-            @JsonProperty("reportType") ReportType reportType) {
-        this.items = items;
-        this.reportType = reportType;
-    }
-    public List<T> getItems() {
-        return items;
-    }
-    public int getTotal() {
-        return items.size();
+    public ReportTypeResourceList(@JsonProperty(ITEMS) List<T> items) {
+        super(items);
     }
+    @Deprecated
     public ReportType getReportType() {
-        return reportType;
+        return (ReportType)getRequestParams().get(REPORT_TYPE);
+    }
```

Comment:
I think I see what you're looking for. Added in ReportTypeResourceListTest line 33 after commit, there are tests below of the serialization of this list to verify it is there in the JSON.